commands/instances.go

```diff
@@ -558,13 +561,27 @@ func Upgrade(ctx context.Context, req *rpc.UpgradeReq, downloadCB DownloadProgre
 
 				// In case uninstall fails tries to rollback
 				if err != nil {
+					logrus.WithError(err).Error("Error updating platform.")
 					taskCB(&rpc.TaskProgress{Message: "Error upgrading platform: " + err.Error()})
 
 					// Rollback
 					if err := pm.UninstallPlatform(latest); err != nil {
+						logrus.WithError(err).Error("Error rolling-back changes.")
 						taskCB(&rpc.TaskProgress{Message: "Error rolling-back changes: " + err.Error()})
 					}
```

Comment:
I think we need to return here in case of rollback and not perform the postinstall